tests/arch/arm64/arm64_high_addresses/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

```suggestion
cmake_minimum_required(VERSION 3.20.0)
```

Need 3.20.0 now (see #37575).